website/source/docs/configuration/index.html.md

```diff
@@ -179,6 +183,22 @@ testing.
 
 - `log_json` `(bool: false)` - Output logs in a JSON format.
 
+- `log_file` `(string: "")` - Specifies the path for logging. If the path
+  does not includes a filename, the filename defaults to "nomad-{timestamp}.log".
+  This setting can be combined with `log_rotate_bytes` and `log_rotate_duration`
+  for a fine-grained log rotation control.
+
+- `log_rotate_bytes` `(int: 0)` - Specifies the number of bytes that should be
+  written to a log before it needs to be rotated. Unless specified, there is no
+  limit to the number of bytes that can be written to a log file.
+
+- `log_rotate_duration` `(duration: "24h")` - Specifies the maximum duration a
+  log should be written to before it needs to be rotated. Must be a duration
+  value such as 30s.
+
+- `log_rotate_max_files` `(int: 0)` - Specifies the maximum number of older log
+  file archives to keep. If 0 no files are ever deleted.
+
```

Comment:
@endocrimes  I pulled in the docs from the logging PR, but I didn't dig into the types.  If you see anyplace I specifically lied, could you heal it?